tests/com/tony/server/WorkerThreadTest.java

```diff
@@ -1,37 +1,98 @@
 package com.tony.server;
 
+import com.tony.server.response.HeadResponse;
+import org.junit.After;
```

Comment:
Intellij won't let me fix it but this type of ordering seems to be consistent throughout the code.